pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -142,17 +142,28 @@ def cmd(self):
         if self.disable_capture:
             cmd.append("-s")
 
-        if self.processes == -1:
-            cmd.append('-n auto')
-            cmd.append('--dist=loadscope')
-        elif self.processes != 0:
-            cmd.append('-n {}'.format(self.processes))
-            cmd.append('--dist=loadscope')
-
-        if not self.randomize:
-            cmd.append('-p no:randomly')
-        if self.eval_attr:
-            cmd.append("-a '{}'".format(self.eval_attr))
+        if self.xdist_ip_addresses:
+            cmd.append('-d')
```

Comment:
I think we still want `--dist=loadscope` here instead of `-d` (which is an abbreviation for `--dist=load`).